src/Components/Alert/Components/Filters/Color.tsx

```diff
@@ -0,0 +1,106 @@
+import { Checkbox, Box, Text, Flex, Spacer } from "@artsy/palette"
+import * as React from "react"
+import styled from "styled-components"
+import { themeGet } from "@styled-system/theme-get"
+import { COLOR_OPTIONS } from "Components/ArtworkFilter/ArtworkFilters/ColorFilter"
+import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
+
+type ColorOption = typeof COLOR_OPTIONS[number]
+
+const ColorSwatch = styled.div`
+  width: ${themeGet("space.2")};
+  height: ${themeGet("space.2")};
+  border-radius: ${themeGet("space.2")};
+  flex-shrink: 0;
+`
+
+const BlackAndWhiteSwatch = styled(ColorSwatch)`
```

Comment:
#superminor - its nice if these css-ish components can be moved to the bottom of the file, since this stuff rarely changes and is mostly noise. Ideally a dev can open the file and see first the very component that the file is named after. So like... 

```
const Color = 
...
const ColorFilterOption = 
...
// everything else